core/services/eth/errors.go

```diff
@@ -93,7 +95,13 @@ var arbitrum = ClientErrors{
 	Fatal:                 arbitrumFatal,
 }
 
-var clients = []ClientErrors{parity, geth, arbitrum}
+var optimism = ClientErrors{
+	// TODO: Test these
```

Comment:
might be worth creating a task for this one

Reply:
I just added the test to this pr